app/enterprise/0.33-x/developer-portal/best-practice.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Kong Developer Portal Best Practices
+book: portal
+chapter: 12
+---
+
+# Kong Developer Portal Best Practices
+
+The Kong Developer Portal is a powerful tool. It's part static website and part interactive API. With a tool of this magnitude, there are countless ways to configure and host that will vary greatly depending on your needs. The following article is a group of recommendations we have found to lead which lead to  easy administrative and top performance. This list is not all inclusive, so if you have a trick you like to use and want to share it with others, please feel free to make a PR on the page, or send us an email at <support@konghq.com> and we can add it for you.
```

Comment:
>this list is not all inclusive

What do you think about:

> This list is not exhaustive

or 

> This list is not fully comprehensive

Reply:
Really good idea. I went  with This list is not exhaustive because I liked how it sounded